activerecord/test/cases/adapters/mysql2/connection_test.rb

```diff
@@ -104,8 +104,8 @@ def test_wait_timeout_as_url
   end
 
   def test_mysql_connection_collation_is_configured
-    assert_equal "utf8_unicode_ci", @connection.show_variable("collation_connection")
-    assert_equal "utf8_general_ci", ARUnit2Model.connection.show_variable("collation_connection")
+    assert_equal "utf8mb4_general_ci", @connection.show_variable("collation_connection")
+    assert_equal "utf8mb4_general_ci", ARUnit2Model.connection.show_variable("collation_connection")
```

Comment:
Using default collation in both connections would lose the purpose this test.
At least one connection should be given non default collation. See https://github.com/rails/rails/commit/d5ad2037455a03261d680d143257ba253d8a7112